src/test/java/org/opensearch/knn/bwc/KNNBackwardsCompatibilityIT.java

```diff
@@ -0,0 +1,164 @@
+/*
+ *  Copyright OpenSearch Contributors
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.bwc;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import org.apache.http.util.EntityUtils;
+import org.opensearch.knn.KNNResult;
+import org.opensearch.knn.index.KNNQueryBuilder;
+import org.opensearch.knn.KNNRestTestCase;
+import org.opensearch.client.Request;
+import org.opensearch.client.Response;
+import org.junit.Assert;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.test.rest.OpenSearchRestTestCase;
+
+public class KNNBackwardsCompatibilityIT extends KNNRestTestCase {
```

Comment:
How does this not get excluded by [this](https://github.com/opensearch-project/k-NN/pull/226/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R356)?